components/UserInput.tsx

```diff
@@ -0,0 +1,67 @@
+'use client'
+
+import { Space_Mono } from '@next/font/google'
+import { useState } from 'react';
+
+const smono = Space_Mono({
+    weight: ['400', '700'],
+    variable: '--inter-font',
+    preload: true,
+    display: 'swap',
+  })
+
+interface UserInputProps {
+    inputAmount: number;
+    value: string;
+    userInput: (value: string) => void
+}
+
+export const UserInput: React.FC<UserInputProps> = ({ inputAmount, value, userInput }) => {
+    const [textAreaValue, setTextAreaValue] = useState(value);
+    
+    return (
+        <form style={{position:'relative'}} >
+              <br/>
+              <textarea 
+                onChange={(e) => {setTextAreaValue(e.target.value)
+                    userInput(e.target.value);
+                }}
+                value={textAreaValue}  
+                id="text" name="text"
+                rows={4} cols={40}
```

Comment:
We may need these to be customizable. Do you know if browser strictly enforce rows and cols or if they can be overwritten with CSS?